CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# CHANGELOG
```

Comment:
We should add 'Update CHANGELOG' to be PR checklist.

Reply:
Yeah, makes sense. Good point.